meinberlin/config/settings/base.py

```diff
@@ -297,7 +297,8 @@
 LOGIN_URL = "account_login"
 LOGIN_REDIRECT_URL = "/"
 
-EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+EMAIL_BACKEND = "django.core.mail.backends.filebased.EmailBackend"
```

Comment:
I'd personally prefer to keep the default as `console`. For testing you can put this in our `local.py` or just don't commit it :)